db/c_test.c

```diff
@@ -2956,6 +2956,51 @@ int main(int argc, char** argv) {
     CheckNoError(err);
   }
 
+  StartPhase("filter_with_prefix_seek");
+  {
+    rocksdb_close(db);
+    rocksdb_destroy_db(options, dbname, &err);
+    CheckNoError(err);
+
+    rocksdb_options_set_prefix_extractor(
+        options, rocksdb_slicetransform_create_fixed_prefix(1));
```

Comment:
Does this prefix extractor need to be deleted to prevent a leak?

Reply:
Please see my response to an earlier comment.